Firestore/Example/Tests/Util/FSTHelpers.mm

```diff
@@ -304,14 +304,25 @@ MaybeDocumentMap FSTTestDocUpdates(const std::vector<MaybeDocument> &docs) {
 
 RemoteEvent FSTTestAddedRemoteEvent(const MaybeDocument &doc,
                                     const std::vector<TargetId> &addedToTargets) {
-  HARD_ASSERT(!doc.is_document() || !Document(doc).has_local_mutations(),
-              "Docs from remote updates shouldn't have local changes.");
-  DocumentWatchChange change{addedToTargets, {}, doc.key(), doc};
+  const std::vector<MaybeDocument> docs{doc};
```

Comment:
Nit: we don't use "shallow" `const` on local variables and function arguments (that is, `const` references and pointers to `const` are okay, but non-pointer non-reference variables shouldn't be `const`). While I personally like `const`, this is inconsistent with the rest of the codebase.

Reply:
That makes sense. Updated.